hack/ci-deploy.sh

```diff
@@ -71,6 +71,11 @@ else
   kubectl_create -n=local-csi-driver -f="${SO_CSI_DRIVER_PATH}"
 fi
 
+if [[ -n "${SO_SCYLLACLUSTER_STORAGECLASS_NAME}" ]]; then
+  yq e --inplace ".spec.datacenter.racks[0].storage.storageClassName = \"${SO_SCYLLACLUSTER_STORAGECLASS_NAME}\"" "${DEPLOY_DIR}/manager/50_scyllacluster.yaml"
```

Comment:
```suggestion
  yq e --inplace '.spec.datacenter.racks[0].storage.storageClassName = env(SO_SCYLLACLUSTER_STORAGECLASS_NAME)' "${DEPLOY_DIR}/manager/50_scyllacluster.yaml"
```

Reply:
why do you think that's better?

Reply:
it's native and properly quotes the value (e.g. nested quotes)

Reply:
ok, done